x-pack/test/functional_with_es_ssl/apps/cases/group1/view_case.ts

```diff
@@ -377,52 +396,125 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         expect(await historyBadge.getVisibleText()).equal('3');
       });
 
-      it('filters by comment successfully', async () => {
-        await testSubjects.click('user-actions-filter-activity-button-comments');
+      it('sorts by newest first successfully', async () => {
+        await testSubjects.click('user-actions-filter-activity-button-all');
+
+        const AllBadge = await find.byCssSelector(
+          '[data-test-subj="user-actions-filter-activity-button-all"] span.euiNotificationBadge'
+        );
+
+        expect(await AllBadge.getVisibleText()).equal('4');
+
+        const sortDesc = await find.byCssSelector(
+          '[data-test-subj="user-actions-sort-select"] [value="desc"]'
+        );
+
+        await sortDesc.click();
+
+        await header.waitUntilLoadingHasFinished();
+
+        const userActionsLists = await find.allByCssSelector(
+          '[data-test-subj="user-actions-list"]'
+        );
+
+        const actionList = await userActionsLists[0].findAllByClassName('euiComment');
+
+        expect(await actionList[0].getAttribute('data-test-subj')).contain('status-update-action');
+      });
+
+      it('shows more actions on button click', async () => {
+        await cases.common.selectSeverity(CaseSeverity.LOW);
```

Comment:
yes, I am adding different user actions so that User actions reach to more than 20 and then Show more button is visible. No it will automatically `reduce comments and show load more button` when 21st comment is added.

I will create separate functionality to generate user actions.